dali/base/dautils.hpp

```diff
@@ -140,6 +140,7 @@ class da_decl CDfsLogicalFileName
     bool isExpanded() const;
     void expand(IUserDescriptor *user);
     void normalizeName(const char * name, StringAttr &res, bool strict);
+    bool normalizeExternal(const char * name, bool strict);
```

Comment:
If it were public, it should return the normalizeExternal as normalizeName does, which it should do anyway for consistency.